data/abhay-raj19/2022-12-10-Google-Devfest'22.json

```diff
@@ -0,0 +1,13 @@
+{
+  "isVirtual": false,
+  "isInPerson": true,
+  "color": "green",
+  "name": "Google Devfest kanpur",
+  "description": "In this devfest,We had met many young Devlopers enlightining there carrer in Emerging Tech Domains,got hands into those teck-Stack,got an awesome Experience",
```

Comment:
Hope you don't mind the below:

`"In this devfest,We had met many young Devlopers enlightining there carrer in Emerging Tech Domains,got hands into those teck-Stack,got an awesome Experience"`

to:

`"At this devfest, We met many young Devlopers enlightining their career in Emerging Tech Domains, got hands-on into those tech stacks, and had an awesome experience"`

reads a lot better :smile:

Reply:
I had done the necessary changes.👍 pls see to it.